### PR TITLE
Adds new felinid-only midround event that simulates the actions of a cat

### PR DESCRIPTION
**Pull request by maxion12345**
# Document the changes in your pull request
4xAY2WE

Adds a mid-round objective that makes a felinid leave a dead rat outside the door of their department head to show their appreication.
HAFmKNb

# Why is this good for the game?
felinid more realistic high rp

# Testing

jyBZNdy

# Spriting
n/a

# Wiki Documentation

add to random event page
has same conditions as spontaneous apprendictis
# Changelog

:cl:  
rscadd: Added new felinid-specific midround event that makes them leave a dead rat outside the door of their department head
/:cl:

### CODE REVIEW

**Comment by RG4ORDR** [comment]
Holy shit.
Merge this.

**Comment by azzzertyy** [comment]
felinids already do this

**Comment by Addust** [comment]
what if they're captain? are they supposed to just leave one in fucking centcom?

File: code/modules/events/felinidrat.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+			continue
+		if(!iscatperson(H))
+			continue
+		var/foundAlready = FALSE
+		for(var/datum/brain_trauma/hypnosis/A in H.get_traumas())
+			foundAlready = TRUE
+			break
+		if(foundAlready)
+			continue
+		H.gain_trauma(/datum/brain_trauma/hypnosis, TRAUMA_RESILIENCE_SURGERY, "You feel a need to show how much you appreicate your department head. Leave a dead rat outside the door to their office as a gift!")
```
Comment (ktlwjec0) [inline code/modules/events/felinidrat.dm]: appreciate

**Comment by MajManatee** [comment]
Couldnt it be "Bring them a dead rat" rather than "leave it outside their door"? Since most people wont blink twice at a rat on the floor, but one handed to them raises some eyebrows.

**Comment by maxion12345** [comment]
> Couldnt it be "Bring them a dead rat" rather than "leave it outside their door"? Since most people wont blink twice at a rat on the floor, but one handed to them raises some eyebrows.

this is the kind of feedback I like to see fellas

**Comment by maxion12345** [comment]
> what if they're captain? are they supposed to just leave one in fucking centcom?

fax them a photo of a dead rat

**Comment by maxion12345** [comment]
XG2oTYD

https://github.com/yogstation13/Yogstation/assets/23666142/eb39f859-95c9-46d4-a34d-19873940a016

**Review by MajManatee — APPROVED** [review]
I like the part that it announces it to ghosts

**Comment by Marmio64** [comment]
Why does this have 20 weight? It's going to suck up all the events that actually do fun/cool stuff.

**Comment by maxion12345** [comment]
> Why does this have 20 weight? It's going to suck up all the events that actually do fun/cool stuff.

i copied from spontaneous appendicitis because that seems to happen occasionally but its not so rare like the aurora thing that you kinda forget about it existing until it happens
like the dust events

**Comment by azzzertyy** [comment]
> > Why does this have 20 weight? It's going to suck up all the events that actually do fun/cool stuff.
> 
> i copied from spontaneous appendicitis because that seems to happen occasionally but its not so rare like the aurora thing that you kinda forget about it existing until it happens like the dust events

the issue with this in general is that its clogging up the midround system (something already incredibly clogged) with yet another useless midround that doesn't affect anything really. if this was on a different system id be fine with it, but it being just a normal midround makes it hamper gameplay directly.

**Comment by Hislyna** [comment]
As funny as this is, it has been denied by council. 
t4vL3cb

**Comment by MajManatee** [comment]
Hisa has a wierd ass discord background + cowbot said nuh uh

**Pull request closed without merging**